test/org/sagebionetworks/bridge/DefaultStudyBootstrapperTest.java

```diff
@@ -3,35 +3,46 @@
 import static org.junit.Assert.assertEquals;
 import static org.mockito.Mockito.*;
 
+import javax.annotation.Resource;
+
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
 import org.mockito.ArgumentCaptor;
+import org.sagebionetworks.bridge.config.BridgeTestSpringConfig;
 import org.sagebionetworks.bridge.exceptions.EntityNotFoundException;
 import org.sagebionetworks.bridge.models.studies.PasswordPolicy;
 import org.sagebionetworks.bridge.models.studies.Study;
 import org.sagebionetworks.bridge.services.StudyService;
 
 import com.google.common.collect.Sets;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
+@ContextConfiguration(classes = {BridgeTestSpringConfig.class})
+@RunWith(SpringJUnit4ClassRunner.class)
```

Comment:
We should try to avoid setting up the Spring context in unit tests when we can avoid it. In this case, it would be better to mock out DynamoInitializer and TableCreator.